autoload/neomake.vim

```diff
@@ -210,7 +219,7 @@ function! neomake#GetEnabledMakers(...) abort
     return filter(keys(union), 'union[v:val] ==# l')
 endfunction
 
-function! neomake#Make(options) abort
+function! s:Make(options) abort
```

Comment:
I wish we could think of a better name for this function. I have no suggestions, though, so this will probably stick.